src/handlers/assign/auto.ts

```diff
@@ -18,43 +18,45 @@ export const checkPullRequests = async () => {
 
   // Loop through the pull requests and assign them to their respective issues if needed
   for (const pull of pulls) {
-    const linkedIssue = await gitLinkedIssueParser({
+    const linkedIssues = await gitLinkedIssueParser({
       owner: payload.repository.owner.login,
       repo: payload.repository.name,
       pull_number: pull.number,
     });
 
-    // if pullRequestLinked is empty, continue
-    if (linkedIssue == "" || !pull.user || !linkedIssue) {
-      continue;
-    }
+    for (const linkedIssue of linkedIssues) {
+      // if pullRequestLinked is empty, continue
+      if (linkedIssue == "" || !pull.user || !linkedIssue) {
+        continue;
+      }
 
-    const connectedPull = await getPullByNumber(context, pull.number);
+      const connectedPull = await getPullByNumber(context, pull.number);
 
-    // Newly created PULL (draft or direct) pull does have same `created_at` and `updated_at`.
-    if (connectedPull?.created_at !== connectedPull?.updated_at) {
-      logger.debug("It's an updated Pull Request, reverting");
-      continue;
-    }
+      // The new PR, whether it's in draft or in direct form, it has identical `created_at` and `updated_at` timestamps.
+      if (connectedPull?.created_at !== connectedPull?.updated_at) {
+        logger.debug("Skipping because it's not a new PR");
+        continue;
+      }
 
-    const linkedIssueNumber = linkedIssue.substring(linkedIssue.lastIndexOf("/") + 1);
+      const linkedIssueNumber = linkedIssue.substring(linkedIssue.lastIndexOf("/") + 1);
 
-    // Check if the pull request opener is assigned to the issue
-    const opener = pull.user.login;
+      // Check if the pull request opener is assigned to the issue
+      const opener = pull.user.login;
 
-    const issue = await getIssueByNumber(context, +linkedIssueNumber);
-    if (!issue?.assignees) continue;
+      const issue = await getIssueByNumber(context, +linkedIssueNumber);
+      if (!issue?.assignees) continue;
```

Comment:
If there's no assignee, we don't need to assign the opener of the linked PR to the issue?

Reply:
It doesn't mean there is no assinee. 
It just wants to make sure if `assinees` field exists on the issue not if(issue?.assignees.length === 0)

Reply:
Sounds like the same thing.  No assignee means the `assignees` field doesn't exist, right?

Reply:
No assignee means assignees field is empty array, not undefined.

Reply:
this if statement will only execute if `assignees` is undefined or null, but not if its empty array. github should always send empty array if there's no assignees but if they don't send the field, this could possibly be a bug.

Reply:
Exactly. we need to go on,  to assign the author of the linked PR to the issue if assignees is empty array.
If it is undefined or null, it should stop.